superdesk/io/registry.py

```diff
@@ -115,3 +118,29 @@ def service(service_id):
         return ListCursor(
             [service(_id) for _id in registered_feeding_services]
         )
+
+
+def register_webhook(provider_name, webhook_name, webhook_service=None):
```

Comment:
@petrjasek provider id is not visible in ingest setting UI. As we have to construct URL manually, I'll use provider name for now.
In a future step, when we change the UI, I think a webhook URL should be automatically generated (with auth key) and displayed, and provider id should then be used indeed.